docs/quickstart/avalanche-faucet.md

```diff
@@ -0,0 +1,428 @@
+# Avalanche Faucet
```

Comment:
```suggestion
# Avalanche Subnet Faucet
```
I would suggest to change the title to this and move it under Subnets section. Also move `## Using the Faucet` to the bottom this doc.